code/game/gamemodes/cult/runes.dm

```diff
@@ -38,11 +39,30 @@ To draw a rune, use an arcane tome.
 
 	var/req_keyword = 0 //If the rune requires a keyword - go figure amirite
 	var/keyword //The actual keyword for the rune
+	var/image/ai_image
 
 /obj/effect/rune/New(loc, set_keyword)
 	..()
 	if(set_keyword)
 		keyword = set_keyword
+	runes += src
+
+	var/obj/effect/decal/cleanable/blood/splatter/s = new(loc)
```

Comment:
I don't think you even need the ` = /obj/effect/decal/cleanable/blood/splatter` part, do you?